packages/paste-customization/__tests__/CustomizationProvider.test.tsx

```diff
@@ -51,7 +51,7 @@ const BaseThemeConsumerExample: React.FC<React.PropsWithChildren<BaseThemeConsum
 
 describe('CustomizationProvider', () => {
   it('should render without crashing', (): void => {
-    const root = ReactDOM.createRoot(document.querySelector('div') as HTMLElement);
+    const root = ReactDOM.createRoot(document.createElement('div') as HTMLElement);
```

Comment:
question: why this change? querySelector should be finding a div on the page but createElement creates an entirely new div, so the behavior is different

Reply:
We currently have createElement on main - https://github.com/twilio-labs/paste/blob/main/packages/paste-customization/__tests__/CustomizationProvider.test.tsx#L54

I'll double check if a div is on the page but they were failing when we ran the tests with the querySelector

Reply:
Oh if this is just reverting to something we're doing in Paste that works. I guess this was introduced by accident somewhere earlier in this upgrade?

Reply:
Yes exactly, looks like this was an accident 🙂